### PR TITLE
test: interview-session loadersの統合テストを追加

### LINKED ISSUE #373: Phase 3: Server loaders / actions の統合テストを追加
## 概要

テストカバレッジ80%達成の Phase 3。Server Components 向けのデータ取得関数（loaders）と Server Actions のテストを追加する。

## 背景

loaders はDBからデータを取得してServer Componentsに渡す関数。actions はフォーム送信等のサーバーサイド処理。いずれも Supabase クライアントに依存するが、ローカル Supabase に対して実行可能。

### interview-session/server/loaders/ （~214行、現在0%）

| ファイル | 行数 | テスト内容 |
|----------|------|-----------|
| `load-interview-messages.ts` | 26 | セッションIDでメッセージ一覧を取得 |
| `load-interview-session-by-id.ts` | 59 | セッションID + ユーザー所有権検証 |
| `load-interview-session.ts` | 27 | アクティブセッションの取得 |
| `load-or-create-interview-session.ts` | 52 | 既存セッション取得 or 新規作成 |
| `load-interview-chat.ts` | 50 | セッション + メッセージ + 設定の一括取得 |

### interview-session/server/actions/ （~60行、現在0%）

| ファイル | 行数 | テスト内容 |
|----------|------|-----------|
| `create-interview-session.ts` | 32 | 新規セッション作成アクション |
| `delete-interview-session.ts` | 28 | セッション削除アクション |

### bills/server/loaders/ （推定~300行、現在0%）

| ファイル | テスト内容 |
|----------|-----------|
| 法案一覧取得 | ページネーション・フィルタリングのテスト |
| 法案詳細取得 | 関連データの結合取得テスト |

### chat/server/ 関連

| ファイル | テスト内容 |
|----------|-----------|
| loaders | チャット履歴の取得テスト |
| actions | チャットセッション管理のテスト |

## テスト方針

- loaders は `"server-only"` を import しているため、統合テスト設定（`vitest.integration.config.mts`）で実行
- テストデータは `tests/supabase/utils.ts` の既存ヘルパーを使用
- Server Actions は `"use server"` の関数だが、テストからは普通の関数として呼び出し可能
- `cookies()` や `redirect()` に依存するものはモックが必要 → vi.mock で対応

```typescript
// cookies/redirect のモック例
vi.mock("next/headers", () => ({
  cookies: () => ({
    get: () => undefined,
    set: () => {},
  }),
}));
vi.mock("next/navigation", () => ({
  redirect: (url: string) => {
    throw new Error(`REDIRECT:${url}`);
  },
}));
```

## テストファイル配置

ソースファイルと co-locate:

```
web/src/features/interview-session/server/loaders/
├── load-interview-messages.ts
├── load-interview-messages.integration.test.ts        ← NEW
├── load-interview-session-by-id.ts
├── load-interview-session-by-id.integration.test.ts   ← NEW
├── load-or-create-interview-session.ts
└── load-or-create-interview-session.integration.test.ts ← NEW

web/src/features/interview-session/server/actions/
├── create-interview-session.ts
├── create-interview-session.integration.test.ts       ← NEW
├── delete-interview-session.ts
└── delete-interview-session.integration.test.ts       ← NEW
```

## 推定効果

- 追加カバー: **~1,500行**
- Phase 3 完了後の推定カバレッジ: **~52%**

## 検証方法

```bash
npx supabase start
pnpm --filter web test:integration
pnpm typecheck
pnpm lint
```

## 関連

- #370 カバレッジ除外設定
- #371 Phase 1: 純粋関数テスト
- #372 Phase 2: LLM Service テスト

### PR DESCRIPTION
**Pull request by muraikenta**
## 概要

Phase 3 (#373) の一部として、`web/src/features/interview-session/server/loaders/` の統合テストを追加します。

## 追加テスト

| ファイル | テスト数 | 主な検証内容 |
|---------|---------|------------|
| `get-interview-messages.integration.test.ts` | 5 | 所有者・未認証・別ユーザー・空・順序 |
| `get-interview-session-by-id.integration.test.ts` | 5 | 所有者・未認証・別ユーザー・存在しないID・完了済み |
| `get-interview-session.integration.test.ts` | 5 | アクティブ・未認証・存在なし・完了済み・別ユーザー |
| `get-latest-interview-session.integration.test.ts` | 6 | active/completed/未認証/存在なし/アーカイブ/最新優先 |
| `initialize-interview-chat.integration.test.ts` | 2 | 既存セッション利用・新規セッション作成 |

合計: **23テストケース**

## 実装方針

loaders は `getChatSupabaseUser()` (Next.js `cookies()` 依存) を通じて認証を行うため、
テスト環境では `vi.mock("@/features/chat/server/utils/supabase-server")` で差し替え、
DB レイヤー（Supabase）は実際のローカルインスタンスに接続して動作を検証する統合テスト構成を採用。

`initialize-interview-chat` では LLM 呼び出しを行う `generateInitialQuestion` も
`vi.mock` で差し替えて外部 AI 依存を排除。

## テスト実行結果

```
pnpm lint        ✓ No fixes applied
pnpm typecheck   ✓ No errors
pnpm --filter web test:integration  ✓ 35 passed (8 test files)
```

## 関連

- Closes #373（次の #373-2 完了時）

## Summary by CodeRabbit

* **Tests**
  * Added comprehensive integration tests covering interview sessions, message retrieval, latest-session selection, and chat initialization to improve reliability across auth and edge cases.
* **Bug Fixes**
  * Ensured deterministic message ordering when timestamps match to prevent inconsistent message sequences.
* **Refactor**
  * Introduced dependency injection for authentication and chat initialization flows to improve robustness and testability without altering user-facing behavior.

### CODE REVIEW

File: web/src/features/interview-session/server/loaders/get-interview-messages.integration.test.ts [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    error: null,
+  });
+}
+
+function mockUnauthenticated() {
+  vi.mocked(getChatSupabaseUser).mockResolvedValue({
+    data: { user: null },
+    error: new Error("Not authenticated") as never,
+  });
+}
```
Comment (muraikenta) [inline web/src/features/interview-session/server/loaders/get-interview-messages.integration.test.ts]: mockは極力使わず、DIに寄せたい

File: web/src/features/interview-session/server/loaders/initialize-interview-chat.integration.test.ts [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  createChatSupabaseServerClient: vi.fn(),
+}));
+
+// generateInitialQuestion はLLM依存のため差し替える
+vi.mock(
+  "@/features/interview-session/server/services/generate-initial-question",
+  () => ({
+    generateInitialQuestion: vi.fn().mockResolvedValue(null),
+  })
+);
```
Comment (muraikenta) [inline web/src/features/interview-session/server/loaders/initialize-interview-chat.integration.test.ts]: LLM依存は他テストでMockModelを使ってうまくやってたはず
